### ISSUE TITLE
Non-MD5 passwords for superuser appear as `changed`

### ISSUE THREAD

**Issue by multani** [edgelaboratories/terraform-provider-postgresql#5]
This state:
```
resource postgresql_role "xxx" {
  name      = "xxx"
  login     = true
  password  = "xxx"
  superuser = true
}
```

always produces the following plan:
```
  ~ postgresql_role.xxx
      password: <sensitive> => <sensitive> (attribute changed)
```

**Comment by multani** [comment]
This should already be fixed in https://github.com/terraform-providers/terraform-provider-postgresql/pull/44/

**Comment by cyrilgdn** [comment]
Fixed by #44 and #54. Merged in [internal-release](https://github.com/edgelaboratories/terraform-provider-postgresql/tree/internal-release)
